workspaces/applications.md

```diff
@@ -82,6 +82,13 @@ apps:
 - A health check _must_ report healthy for you to access the application.
 - If you specify both the HTTP and Exec health checks, Coder prioritizes HTTP.
 
-You can include the applications specification file in your
-[workspace image](../images/writing.md). You an also modify the file via
-[personalization](./personalization.md) scripts.
+## Sample usage
+
+Coder offers an
+[image](https://github.com/cdr/enterprise-images/tree/91ef8f521b2275783fed54b27052cc544153cd99/images/vnc)
```

Comment:
I think we should refer to the Docker Hub image name (i.e. codercom/enterprise-vnc) instead of the image source in the repository, since the source can change